sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/package-info.java

```diff
@@ -3,7 +3,7 @@
 // Code generated by Microsoft (R) AutoRest Code Generator.
 
 /**
- * Package containing the implementations for KeyVaultAccessControlClient. The key vault client performs cryptographic
- * key operations and vault operations against the Key Vault service.
+ * Package containing the implementations for KeyVaultBackupClient. The key vault client performs cryptographic key
```

Comment:
This got changed by Autorest, is there any way of replacing the string with what we used to have?